index.js

```diff
@@ -1008,31 +1053,42 @@ Multi.prototype.send_command = function (command, args, index, cb) {
     });
 };
 
-Multi.prototype.exec = Multi.prototype.EXEC = function (callback) {
+Multi.prototype.exec_transaction = function (callback) {
     var self = this;
+    var len = this.queue.length;
+    var cb;
```

Comment:
That is indeed true. The question here is though: do you want to add another check for a case that will be handled properly anway? The .batch function works differently, as it would otherwise not behave the same as with commands. So I'd say it's fine to not handle that case any special.